README.md

```diff
@@ -583,6 +584,7 @@ Table of Contents
   * [getstream.io](https://getstream.io/) — Build scalable news feeds and activity streams in a few hours instead of weeks, free for 3 million feed updates/month
   * [hasura.io](https://hasura.io/) — Platform to build and deploy app backends fast, free for single node cluster.
   * [iron.io](https://www.iron.io/) — Async task processing (like AWS Lambda) with free tier and 1-month free trial
+  * [linx.software](https://linx.software/) — A low-code tool to easily design, connect and host server-side services into your website or mobile application
```

Comment:
Please add some details about whats free - like what SaaS resources you get in the free tier since the web site just says get started for free its not clear

Reply:
I made some changes to the wording to indicate what's free. 
Thanks